src/components/flyout/deviceDetailFlyout.scss

```diff
@@ -21,6 +21,14 @@
     }
   }
 
+  .btn-group {
+    display: flex;
+    justify-content: flex-end;
+    margin-top: 32px;
+
+    .pcs-btn { height: 30px; }

```

Comment:
Just a note, I think `height: 30px;` on the `.btn-group` will accomplish the same goal without having to style the `.pcs-btn`